pymc3/aesaraf.py

```diff
@@ -146,6 +148,19 @@ def change_rv_size(
         Expand the existing size by `new_size`.
 
     """
+    if new_size is None:
```

Comment:
I think with https://github.com/pymc-devs/pymc3/commit/0226047e36350f91b4562b6d6647e37806436589 we can probably drop this, I put this in as a short-cut but now with the fixed tests we should probably not keep supporting this.